CanvasColorSpaceProposal.md

```diff
@@ -164,43 +159,67 @@ Note: An alternative approach that was considered was to augment the probablySup
 
 #### ImageBitmap
 
-TODO(ccameron): Review this
-ImageBitmap objects are augmented to have an internal color space attribute of type CanvasColorSpace and an internal pixelFormat attribute of type CanvasPixelFormat. The colorSpaceConversion creation attribute also accepts enum values that correspond to CanvasColorSpace values. Specifying a CanvasColorSpace value results in a conversion of the image to the specified color space.
+ImageBitmap objects (unless created with ``colorSpaceConversion="none"``) should keep track of their internal color space, and should store their contents at highest fidelity possible, subject to implementation limitations.
 
 #### ImageData
 
-TODO(ccameron): Review this
-
-IDL
+Add the following types to be used by `ImageData`.
 <pre>
-
-enum ImageDataStorageType {
+enum ImageDataEncoding {
   "uint8", // default
   "uint16",
   "float32",
 };
-
+dictionary ImageDataSettings {
+  CanvasColorSpaceEnum colorSpace = "srgb";
+  ImageDataStorageType encoding = "uint8";
```

Comment:
This still references the "ImageDataStorageType" data type. See above discussion. I lean toward leaving it as "ImageDataStorageType".

Reply:
Updated all instances to ImageDataStorageType